### PR TITLE
liquidwar: fix build

### PR DESCRIPTION
**Pull request by polygon**
###### Motivation for this change

Configure script would try to use guile-config to determine guile configuration which in turn would attempt to use pkg-config.

Since pkg-config was not available in the build, guile-config would fail and some default options would be used causing guile config to not be properly picked up.

Fixed by adding pkg-config to dependencies.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by 7c6f434c** [comment]
Shouldn't that be in `nativeBuildInputs`?

**Comment by polygon** [comment]
Yes, just noticed. PR is updated, thanks.

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 125480` at 9afb66ad run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>liquidwar</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/build-tools-in-build-inputs.md">build-tools-in-build-inputs</a>

libtool is a build tool so it likely goes to `nativeBuildInputs`, not `buildInputs`.

Near pkgs/games/liquidwar/default.nix:18:3:
```
   |
18 |   buildInputs = [
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/games/liquidwar/default.nix [hunk 1]
```diff
@@ -3,7 +3,8 @@
 , expat, gettext, perl, guile
 , SDL, SDL_image, SDL_mixer, SDL_ttf
 , curl, sqlite, libtool, readline
-, libogg, libvorbis, libcaca, csound, cunit } :
+, libogg, libvorbis, libcaca, csound, cunit
+, pkg-config } :
```
Comment (SuperSandro2000) [inline pkgs/games/liquidwar/default.nix:7]:
```suggestion
, libogg, libvorbis, libcaca, csound, cunit
, pkg-config }:
```

File: pkgs/games/liquidwar/default.nix [hunk 2]
```diff
@@ -25,6 +26,8 @@ stdenv.mkDerivation rec {
     libtool readline
   ];
 
+  nativeBuildInputs = [ pkg-config ];
+
   hardeningDisable = [ "format" ];
```
Comment (SuperSandro2000) [inline pkgs/games/liquidwar/default.nix:31]: Can you take a look at if this can be removed?

**Pull request merged**